### PR TITLE
TL Localization - Chapter 6, section 3 - part 1 (1,992 words)

### PR DESCRIPTION
**Pull request by heywens**

### CODE REVIEW

File: book/06-github/sections/3-maintaining.asc [hunk 1]
```diff
@@ -1,134 +1,133 @@
 [[_maintaining_gh_project]]
-=== Maintaining a Project
+=== Pagpapanatili ng isang Proyekto
 
-Now that we're comfortable contributing to a project, let's look at the other side: creating, maintaining and administering your own project.
+Ngayon na kumportable tayo na nag-aambag sa isang proyekto, tingnan natin ang kabilang panig: paglikha, pagpapanatili at pangangasiwa ng iyong sariling proyekto.
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: `tayo na` could be shortened to `tayong`

File: book/06-github/sections/3-maintaining.asc [hunk 2]
```diff
@@ -1,134 +1,133 @@
...
+.Ang dropdown na ``Bagong repositoryo''.
+image::images/new-repo.png[The ``new repositoryo'' dropdown.]
 
-This takes you to the ``new repository'' form:
+Ito ay nagdadala sa iyo sa form ng ``bagong repositoryo'':
 
-.The ``new repository'' form.
-image::images/newrepoform.png[The ``new repository'' form.]
+.Ang form ng ``bagong repositoryo''.
+image::images/newrepoform.png[The ``new repositoryo'' form.]
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: `new` to `bagong`

File: book/06-github/sections/3-maintaining.asc [hunk 3]
```diff
@@ -1,134 +1,133 @@
...
+Ang HTTPS ay isa ring eksaktong parehong URL na kanilang idikit sa isang browser upang tingnan ang proyekto doon.
 ====
 
-==== Adding Collaborators
+==== Pagdaragdag ng mga Tagapangasiwa
 
-If you're working with other people who you want to give commit access to, you need to add them as ``collaborators''.
-If Ben, Jeff, and Louise all sign up for accounts on GitHub, and you want to give them push access to your repository, you can add them to your project.
-Doing so will give them ``push'' access, which means they have both read and write access to the project and Git repository.
+Kung ikaw ay nagtatrabaho kasama ang ibang tao nga nais mong bigyan ng access sa pag-commit, kailangan mong idagdag sila bilang ``tagapangasiwa''.
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: `nga` to `na`

File: book/06-github/sections/3-maintaining.asc [hunk 4]
```diff
@@ -1,134 +1,133 @@
...
+Kung si Ben, Jeff, and Louise ay nag-sign up ng mga account sa GitHub, at gusto mo silang bigyan ng access sa pag-push sa iyong repositoryo, maaari mo silang idagdag sa iyong proyekto.
+Paggawa nito ay nagbibigay sa kanila ng access sa ``push'', na nangangahulugan na mayroon silang access sa pagbasa at pagsulat sa proyekto at  repositoryo ng Git.
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: - you could change `Paggawa` to `Ang paggawa`
- spacing between `at` and  `repositoryo`

File: book/06-github/sections/3-maintaining.asc [hunk 5]
```diff
@@ -1,134 +1,133 @@
...
 
-Click the ``Settings'' link at the bottom of the right-hand sidebar.
+I-click ang link ng ``Settings'' sa ibaba ng kanang sidebar.
 
-.The repository settings link.
-image::images/reposettingslink.png[The repository settings link.]
+.Ang link ng settings ng repositoryo.
+image::images/reposettingslink.png[The repositoryo settings link.]
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: `[Ang link ng settings ng repositoryo.]`

File: book/06-github/sections/3-maintaining.asc [hunk 6]
```diff
@@ -1,134 +1,133 @@
...
-If you need to revoke access, just click the ``X'' on the right-hand side of their row.
+Pagkatapos piliin ang =``Tagapangasiwa'' mula sa menu sa kaliwang bahagi.
+Pagkatapos, magtipa ng username sa kahon, at i-click ang ``Magdagdag ng tagapangasiwa.''
+Maaari mong ulitin ito nang maraming beses hangga't gusto mo upang magbigay ng access sa lahat ng iyong gusto
+Kung kailangan mong bawiin ang access, i-click lamang ang `` X '' sa kanang bahagi ng kanilang hilera.
 
-.Repository collaborators.
-image::images/collaborators.png[The repository collaborators box.]
+.Repository tagapangasiwa.
+image::images/tagapangasiwa.png[The repositoryo tagapangasiwa box.]
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: I think this needs to be translated `The repositoryo tagapangasiwa box.`

File: book/06-github/sections/3-maintaining.asc [hunk 7]
```diff
@@ -1,134 +1,133 @@
...
 ----
 $ curl http://github.com/tonychacon/fade/pull/1.patch | git am
 ----
 
-===== Collaborating on the Pull Request
+===== Pangangasiwa sa Kahilingan na Pull
 
-As we covered in <<_github#_github_flow>>, you can now have a conversation with the person who opened the Pull Request.
-You can comment on specific lines of code, comment on whole commits or comment on the entire Pull Request itself, using GitHub Flavored Markdown everywhere.
+Sa natalakay sa <<_github#_github_flow>>, maaari ka na ngayong magkaroon ng pag-uusap sa taong nagbukas Kahilingan na Pull.
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: `taong nagbukas ng Kahilingan na Pull`

File: book/06-github/sections/3-maintaining.asc [hunk 8]
```diff
@@ -1,134 +1,133 @@
...
-This is a bit of an advanced trick and we'll go over the details of this a bit more in <<_git_internals#_refspec>>, but it can be pretty useful.
+Kung ikaw ay nakikitungo sa isang *maraming* mga Kahilingan na Pull at hindi nais na magdagdag ng isang bungkos ng mga remote o gumawa ng isang beses na mga pull sa bawat oras, may isang malinis na trick na pinapahintulutan ng GitHub na gawin mo.
+Ito ay isang konting advanced na trick at tatalakayin natin ang mga detalye nito sa <<_git_internals#_refspec>>, ngunit ito ay maaaring kapaki-pakinabang.
 
-GitHub actually advertises the Pull Request branches for a repository as sort of pseudo-branches on the server.
-By default you don't get them when you clone, but they are there in an obscured way and you can access them pretty easily.
+Ang GitHub ay tunay na nag-aanunsiyo ng mga branch ng Kahilingan na Pull para sa isang repositoryo bilang uri ng mga pseudo-branch sa server.Bilang default hindi mo makuha ang mga ito kapag ikaw ay nag-clone, ngunit doon sila sa isang nakakubling paraan at maaari mong madling ma-access ang mga ito.
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: - space between `server.` and `Bilang`
- `madaling`

File: book/06-github/sections/3-maintaining.asc [hunk 9]
```diff
@@ -161,14 +160,15 @@ From https://github.com/libgit2/libgit2
...
 ----
 
-This tells Git, ``Connect to the `origin` remote, and download the ref named `refs/pull/958/head`.''
-Git happily obeys, and downloads everything you need to construct that ref, and puts a pointer to the commit you want under `.git/FETCH_HEAD`.
-You can follow that up with `git merge FETCH_HEAD` into a branch you want to test it in, but that merge commit message looks a bit weird.
-Also, if you're reviewing a *lot* of pull requests, this gets tedious.
+Ito ay nagsasabi sa Git, ``Kumonekta sa remote na `origin`, at i-download ang ref na pinangalanang `refs/pull/958/head`.''
+Masayang sinusunod ng Git, at nagda-download ng lahat na kinakailangan mo sa pagbuo ng ref na iyon, at naglalagay ng isang pointer sa commit na gusto mo sa ilalim ng `.git/FETCH_HEAD`.
+Maaari mong sundan iyon ng `git merge FETCH_HEAD` sa branch na gusto mong suriin, ngunit ang mensahe ng merge commit ay 
+ you want to test it in, but that merge commit ay magiging mukhang kakaiba.
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: remove untranslated line `you want to test it in, but that merge commit`

File: book/06-github/sections/3-maintaining.asc [hunk 10]
```diff
@@ -177,10 +177,10 @@ It should look a bit like this:
...
 ----
 
-That line that begins with `fetch =` is a ``refspec.''
-It's a way of mapping names on the remote with names in your local `.git` directory.
-This particular one tells Git, "the things on the remote that are under `refs/heads` should go in my local repository under `refs/remotes/origin`."
-You can modify this section to add another refspec:
+Ang linyang iyon na nagsisimula sa `fetch =` ay isang ``refspec.''
+Ito'y isang paraan ng pagmamapa ng mga pangalan sa remote sa mga pangalan sa iyong lokal na direktoryo na `.git`.
+Ang partikular na ito ay nagsasabi sa Git, "ang mga bagay sa remote na nasa ilalim ng `refs/heads` ay dapat mapunta sa aking lokal na repositoryo sa ilalim ng `refs/remotes/origin`."
+Maaari mo mabago ang seksiyon na ito upang magdagdag ng iba pang refspec:
```
Comment (josteinfrancois) [inline book/06-github/sections/3-maintaining.asc]: `Maaari mong`

**Review by josteinfrancois — COMMENTED** [review]
Great quality translations @heywens . I have only suggested minor changes on your work. Good job.

File: book/06-github/sections/3-maintaining.asc [hunk 11]
```diff
@@ -1,134 +1,133 @@
...
-If you need to revoke access, just click the ``X'' on the right-hand side of their row.
+Pagkatapos piliin ang =``Tagapangasiwa'' mula sa menu sa kaliwang bahagi.
+Pagkatapos, magtipa ng username sa kahon, at i-click ang ``Magdagdag ng tagapangasiwa.''
+Maaari mong ulitin ito nang maraming beses hangga't gusto mo upang magbigay ng access sa lahat ng iyong gusto
+Kung kailangan mong bawiin ang access, i-click lamang ang `` X '' sa kanang bahagi ng kanilang hilera.
 
-.Repository collaborators.
-image::images/collaborators.png[The repository collaborators box.]
+.Tagapangasiwa ng Repositoryo.
+image::images/tagapangasiwa.png[Ang kahon ng tagapangasiwa ng repositoryo.]
```
Comment (dscho) [inline book/06-github/sections/3-maintaining.asc:57]: @heywens was the translation of the image file really intended? This `.png` does not exist...

**Pull request merged**